src/version.cpp.in

```diff
@@ -1,6 +1,4 @@
-#define DEF_LOKI_VERSION_MAJOR 6
-#define DEF_LOKI_VERSION_MINOR 1
-#define DEF_LOKI_VERSION_PATCH 3
+#include "version.h"
```

Comment:
This is undoing a fair chunk of (good) work that was done upstream to get those versions *out* of a header included everywhere.